docs/docs/cross-origin-errors.md

```diff
@@ -0,0 +1,33 @@
+---
+id: cross-origin-errors
+title: Cross-origin Errors
+permalink: docs/cross-origin-errors.html
+---
+
+> Note:
+>
+> The following section applies only to the development mode of React. Error handling in production mode is done with regular try/catch statements.
+
+In [development mode](https://facebook.github.io/react/docs/optimizing-performance.html), React uses a global `error` event handler to preserve the "Pause on exceptions" behavior of browser DevTools. It also logs errors to the developer console.
+
+If an error is thrown from a [different origin](https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy) the browser will mask its details and React will not be able to log the original error message. This is a security precaution taken by browsers to avoid leaking sensitive information.
+
+If you trust the scripts you're running you can ask the browser to bypass the same-origin policy in a secure way. Doing this can help simplify the development/debugging process. Below are a some common causes of cross-origin errors and ways to address them.
```

Comment:
Proper headers are listed below as 1 of the 2 possible causes.

Maybe something like this?

> You can simplify the development/debugging process by ensuring that errors are thrown with a same-origin policy. Below are some common causes of cross-origin errors and ways to address them.

We can also always tweak this wording later.